magento/magento2-page-builder
# Ignore styles which no longer apply

### Description
When PageBuilder initialises, it moves styles from a <style> tag into `data-{viewport}-styles` attributes. If there are CSS selectors that do not match any elements, an error is thrown:

![Screenshot_2022-03-02_11-17-34](https://user-images.githubusercontent.com/334786/156354682-54fa1422-1edb-4310-a499-eafba92a2633.png)

This pull request fixes this bug.

### Manual testing scenarios
1. Use Magento import tool to set a product description (or other Page Builder-enabled attribute) which contains HTML, including some CSS that does not match any elements. (eg, `<span>Sample text</span><style>#not-found { color: red; }</style>`)
1. Navigate to this product in the admin.
1. Witness that Page Builder fails to initialise for this field, and saving the product is impossible (due to a different JavaScript error which stems from the same root cause).
1. Apply the changes in this pull request.
1. Witness that Page Builder initialises as expected, and it is possible to save the product.

### Checklist


### Resolved issues:
1. [x] resolves magento/magento2-page-builder#863: Ignore styles which no longer apply
